Source/Core/Common/BitUtils.h

```diff
@@ -4,6 +4,7 @@
 
 #pragma once
 
+#include <array>
```

Comment:
Is this include actually necessary?  I'm not completely familiar with the rules for templates but you're not directly using `std::array` in the new things in `BitUtils`.

Reply:
The template still needs to know what an `std::array` is, and from what I can tell forward-declaring things in the `std` namespace is not allowed by the spec.

Reply:
~~Won't it know at the point of use?  Doesn't all it need to know is that there is a data() function?~~

Oh, I missed the `typename Container = std::array<...>` bit, never mind.  I think that would require the import.